src/services/config.ts

```diff
@@ -53,8 +53,9 @@ class Config {
   }
 
   getLocalConfig() {
+    const basePath = process.env.BASE_URL || './'
```

Comment:
The last question: is it really necessary to have a default value here? I mean this part ` || './'`.

I suppose the `BASE_URL` variable always has a non-null value (at least `/` by default). But don't forget I'm not a Vue developer so I'm just asking you.

Reply:
Someone could set it to `null` in the config file, I guess. I'm no expert on Vue or webpack either ;)

Reply:
They will have many problems in this case and the bad config path is the smallest of them, though :)

So do you expect that I test this change before it will be merged? Or after, maybe.

Reply:
I'm happy to merge this, as is.